content/en/tracing/setup_overview/custom_instrumentation/dotnet.md

```diff
@@ -19,6 +19,9 @@ further_reading:
     - link: 'tracing/visualization/'
       tag: 'Documentation'
       text: 'Explore your services, resources, and traces'
+    - link: 'https://github.com/DataDog/dd-trace-dotnet/tree/master/samples'
+      tag: 'GitHub'
+      text: 'Examples of custom instrumentation'
```

Comment:
I like the idea of linking this here, but none of this are example of custom instrumentation. Two example show how to set up docker images (Linux and Windows) and the other is about automatic trace id injection into logs. No sure what to use for the `text` field, though.

Reply:
can we call them ".net code samples"?

Reply:
That works for me. Let's check with @jaycdave88.

Reply:
The setup pages have this link already, but it's also called `Examples of custom instrumentation` there:

```
/content/en/tracing/setup_overview/setup/dotnet-framework.md
/content/en/tracing/setup_overview/setup/dotnet-core.md
```

Reply:
Thanks for the flag - looks good to me. Thanks everyone